pkg/services/ngalert/api/tooling/definitions/alertmanager.go

```diff
@@ -105,6 +116,51 @@ import (
 //       200: Ack
 //       400: ValidationError
 
+// swagger:model
+type TestReceiversConfig struct {
+	Receivers []*PostableApiReceiver `yaml:"receivers,omitempty" json:"receivers,omitempty"`
+}
```

Comment:
The body parameter does not appear in the spec

```suggestion
// swagger:parameters RoutePostTestReceivers
type TestReceiversConfigParams struct {
	// in:body
	Config TestReceiversConfig
}

// swagger:model
type TestReceiversConfig struct {
	Receivers []*PostableApiReceiver `yaml:"receivers,omitempty" json:"receivers,omitempty"`
}
```

without the suggestion:

<img width="617" alt="Screenshot 2021-08-06 at 4 39 23 PM" src="https://user-images.githubusercontent.com/1632407/128519082-750d668d-bb02-4dac-9c69-ce5a318246d0.png">

after the suggestion:
<img width="622" alt="Screenshot 2021-08-06 at 4 37 43 PM" src="https://user-images.githubusercontent.com/1632407/128519206-320c7baa-e73f-432b-b066-90467c03aba4.png">